Library/Formula/libexosip.rb

```diff
@@ -1,16 +1,18 @@
-require 'formula'
-
 class Libexosip < Formula
-  homepage 'http://www.antisip.com/as/'
-  url 'http://download.savannah.gnu.org/releases/exosip/libeXosip2-3.6.0.tar.gz'
-  sha1 'a53d699208a22a4edf2d239e1dc3a2f4c1fee0d2'
+  homepage "http://www.antisip.com/as/"
+  url "http://download.savannah.gnu.org/releases/exosip/libeXosip2-4.1.0.tar.gz"
+  sha256 "3c77713b783f239e3bdda0cc96816a544c41b2c96fa740a20ed322762752969d"
 
-  depends_on 'pkg-config' => :build
-  depends_on 'libosip'
+  depends_on "pkg-config" => :build
+  depends_on "libosip"
+  depends_on "openssl"
 
   def install
+    # http://growingshoot.blogspot.com/2013_02_01_archive.html
+    ENV.append "LDFLAGS", "-framework CoreFoundation -framework CoreServices "\
+                          "-framework Security"
```

Comment:
https://savannah.nongnu.org/support/?group=exosip

Looks like no. I'll try to submit something.